### PR TITLE
FLY-8289, FLY-8242,FLY-8288, FLY-8289, FLY-8291, FLY-8502,FLY-8583,FL…

### PR DESCRIPTION
**Pull request by VenkatIc-Kore**
…Y-9047 --- bugs fixed

1. SDK: Change Loading skeleton for top-down interface -done
2. SDK: Add 'answerSearch' Flag in search API--done
3. SDK: Image answer snippet added--done
4. Answer Debugger: Everytime Debug icon should be visible-done
5. UI for topdown search experience changes when runtime debugger opened (1.searchbar width increased,
2.close button changed to cross btn and which is moved to top right corner, 3.filter are hiding,
4.runtime debuger loading skeleton changed)

### CODE REVIEW

**Review by BharadwajAd-Kore — APPROVED** [review]
Looks Good.

**Review by SunilSi-kore — APPROVED** [review]
Approved

File: src/components/search/findly-sdk.js [hunk 1]
```diff
@@ -23728,7 +23707,7 @@ if(res?.graph_answer?.payload?.center_panel){
     let title = res?.graph_answer?.payload?.center_panel?.data[0]?.snippet_title?helpers.convertMDtoHTML(res?.graph_answer?.payload?.center_panel?.data[0]?.snippet_title) : '';
     snippetObj = {'title':title,
     'answer':listSnippetData, page_url:res?.graph_answer?.payload?.center_panel?.data[0]?.url,
-    'source':res?.graph_answer?.payload?.center_panel?.data[0]?.source_title || res?.graph_answer?.payload?.center_panel?.data[0]?.source,
+    'source':undefined, //res?.graph_answer?.payload?.center_panel?.data[0]?.source_title || res?.graph_answer?.payload?.center_panel?.data[0]?.source,
```
Comment (BharadwajAd-Kore) [inline src/components/search/findly-sdk.js:23710]: @VenkatIc-Kore, is there any reason for changing the 'source' property to undefined?
CC: @SunilSi-kore

**Review by BharadwajAd-Kore — APPROVED** [review]
Looks good.

**Pull request merged**